lua/autorun/chatsound-minecraft-subtitles.lua

```diff
@@ -0,0 +1,205 @@
+
+if SERVER then
+	resource.AddSingleFile( 'resource/fonts/MinecraftRegular.ttf' )
+	return
+end
+
+local FONT_NAME = 'ChatsoundMinecraftSubtitles'
+local FONT_SCALE = 12
+local MIN_FONT_SIZE = 1
+local MAX_FONT_SIZE = 8
+
+local enabledCV = CreateClientConVar( 'chatsounds_minecraft_subtitles_enable', 0, true, false, 'Toggle Minecraft-like subtitles for chatsounds', 0, 1 )
+local sizeCV = CreateClientConVar( 'chatsounds_minecraft_subtitles_size', 2, true, false, 'Sets Minecraft subtitles size', MIN_FONT_SIZE, MAX_FONT_SIZE )
+
+for scale = MIN_FONT_SIZE, MAX_FONT_SIZE do
```

Comment:
don't create different fonts for different scales or at least nowhere near this many